antsibull/data/docsite/list_of_collections.rst.j2

```diff
@@ -10,3 +10,11 @@ These are the collections with docs hosted on `docs.ansible.com <https://docs.an
 {% for name in collections | sort %}
 * :ref:`@{ name }@ <plugins_in_@{ name }@>`
 {% endfor %}
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+{% for namespace in namespaces | sort %}
+    @{ namespace }@/index
+{% endfor %}
```

Comment:
Would it be more intuitive if we combined these lists instead of leaving them separate?

Reply:
You mean, having a map `namespace name -> list of collection names`, and doing a double iteration to list the collection names, and a simple iteration to list the namespaces? If we would have the data in this form already, I think it would be ok, but computing this just to have one less input to this template seems a bit excessive, since the template also gets a bit more complicated (double loop for collection names instead of a single loop as it is now).

Reply:
I mean that the output that the user reads seems more intuitive like this:

``` rst
* `community <community/index>`_
  *  :ref:`crypto <plugins_in_community.crypto>`_
  *  :ref:`general <plugins_in_community.general>`_
* `containers <containers/index>`_
  * :ref:`podman <plugins_in_comtainers.podman>`_
 ``` 

Possibly with the visible collection name including the namespace again?  (ie: community.general) I always call it "community.general", so maybe the duplication of names is mroe intuitive.

Reply:
Ah. Hmm, good question. If we do this, I would include the namespace again.

I really have no preference on this right now.

Reply:
We could of course also do such a change in a follow-up PR. Since we have no mechanism where users can supply their own templates, adding `namespaces` now and later removing and replacing it by another list isn't a breaking change for user templates.